lib/__fixtures__/curriculum-helpers-remove-white-space.ts

```diff
@@ -4,11 +4,11 @@ This string sentence has various white spaces characters:
 \t* This line has several preceding white space characters.`;
 
 const stringWithWhiteSpaceCharsRemoved =
-  'Thisstringsentencehasvariouswhitespacescharacters:*Thislinestartswithatabcharacter.*Thislinehasseveralprecedingwhitespacecharacters.';
+  "Thisstringsentencehasvariouswhitespacescharacters:*Thislinestartswithatabcharacter.*Thislinehasseveralprecedingwhitespacecharacters.";
```

Comment:
Open to changing it - I just loaded in the defaults haha

Reply:
_For no reason whatsoever_, I prefer double quotes. But, It would be completely unnecessary to change the main repo to use them. Keeping the repos consistent does not matter so long as the default `ctrl + s` formats things.

Reply:
I agree. I find double quotes to be more concise.